frontend/src/queries/schema.ts

```diff
@@ -204,6 +204,7 @@ export interface HogQLQueryModifiers {
     personsJoinMode?: 'inner' | 'left'
     bounceRatePageViewMode?: 'count_pageviews' | 'uniq_urls'
     sessionTableVersion?: 'auto' | 'v1' | 'v2'
+    propertyGroupsMode?: 'enabled' | 'disabled'
```

Comment:
Representing a boolean as an Enum reads a bit silly here, but there is a good reason this is the way it is: this is to support adding a third value that enables property groups with expression rewriting for performance reasons (#24381) without adding yet another modifier or needing to change the type of this field (since changing it could cause problems for any teams that had values set explicitly in `Team.modifiers`.)

I left `auto` _out_ of this field (in contrast with some of the above values) since this field is already nullable. I also [left this out of `set_default_modifier_values`](https://github.com/PostHog/posthog/blob/c9091d9fea0877675c966fe708081616c64ed336/posthog/hogql/modifiers.py#L36-L57), since adding new values there invalidates the entire query cache (see b2050df0d0ceebc0ca8c139e0da4b9a51e1d8b83.)